js/store.js

```diff
@@ -192,35 +205,26 @@ const store = {
     }
   },
 
-  getThirdParties(hostname) {
-    if (!hostname) {
-      throw new Error('getThirdParties requires a valid hostname argument');
-    }
-
-    const firstParty = this.getWebsite(hostname);
-    if ('thirdPartyHostnames' in firstParty) {
-      return firstParty.thirdPartyHostnames;
-    }
-
-    return null;
-  },
-
   setWebsite(hostname, data) {
     const websites = clone(this._websites);
```

Comment:
Just putting `await` infront of `store.setThirdParty` didn't make a difference for me. Why not?

```javascript
await store.setThirdParty(
        documentUrl.hostname,
        targetUrl.hostname,
        data
      );
```